scripts/known_good.json

```diff
@@ -6,7 +6,7 @@
             "sub_dir": "Vulkan-Headers",
             "build_dir": "Vulkan-Headers/build",
             "install_dir": "Vulkan-Headers/build/install",
-            "commit": "v1.3.261"
+            "commit": "v1.3.262"
         },
         {
             "name": "Vulkan-Utility-Libraries",
```

Comment:
Update the commit for the utility libraries as well. Otherwise it won't get the updated dispatch tables.

Reply:
https://github.com/KhronosGroup/Vulkan-Utility-Libraries/pull/82

Reply:
@mikes-lunarg something is wrong with the 1.3.262 tag for the utility libraries.

https://github.com/KhronosGroup/Vulkan-Utility-Libraries/commits/v1.3.262

Reply:
@ziga-lunarg the tag has been fixed for the utility libraries.

Reply:
Doesn't seem to work

Reply:
I'll take a look.

Reply:
Okay. Something is wrong with the tag still. I see what you mean. Use `b0712dfdb1f38cf707d0295c9fc687635046c713` instead.